### PR TITLE
Repair dead links

### LINKED ISSUE #309: Audit all links
## Context

- Follows https://github.com/libp2p/docs/issues/306 to take the time to check all links and references.
- This should be addressed once majority of the MVP doc changes are shipped (i.e. most of the open PRs are merged).

### PR DESCRIPTION
**Pull request by dchiquito**
Fixes #309 

I used `lychee --verbose --exclude twitter.com ./public/` to check for dead links to fix. `twitter.com` is excluded because [@libp2p](https://twitter.com/libp2p) is not accessible without logging in.

This expands on #337 , which fixed the dead links in the introduction.

### CODE REVIEW

**Comment by p-shahi** [comment]
@dchiquito Thanks for this change, I believe another PR recently merged introduced a conflict. Can you resolve it?

File: content/concepts/contribute/community.md [hunk 1]
```diff
@@ -11,5 +11,5 @@ We love talking about libp2p, and we'd be happy to have you in the mix.
 
 Visit our [discussion forums](https://discuss.libp2p.io) to ask questions about [using libp2p](https://discuss.libp2p.io/c/users), discuss exciting [research](https://discuss.libp2p.io/c/research), and [stay up to date with important announcements](https://discuss.libp2p.io/c/news).
 
-We also often hang out on [Discord](https://discord.com/invite/ipfs) and in the #libp2p-implementers channel in the [Filecoin slack](filecoinproject.slack.com
+We also often hang out on [Discord](https://discord.com/invite/ipfs) and in the #libp2p-implementers channel in the [Filecoin slack](https://filecoinproject.slack.com
 ). While Discord is a great places for live chats, remember that discussions can get lost in a sea of messages. For long-lived discussion & support, [the discussion forums](https://discuss.libp2p.io) are the way to go.
```
Comment (p-shahi) [inline content/concepts/contribute/community.md:15]:
```suggestion
We also hang out in the #libp2p-implementers channel in the [Filecoin slack](https://http://filecoin.io/slack).
```

**Review by p-shahi — APPROVED** [review]
thank you!

**Comment by p-shahi** [comment]
@dchiquito Are you interested in adding this check to CI?

**Comment by dchiquito** [comment]
@p-shahi It's [already there](https://github.com/libp2p/docs/actions/workflows/link-check.yml), actually! It's set to only run on demand, nothing but a manual intervention triggers it at the moment.

I can put in a PR to disable twitter.com checks and run it on any pushes, since it doesn't seem to have much visibility at the moment.

**Comment by dchiquito** [comment]
It turns out I was not using lychee correctly and it was allowing a bunch of false negatives. I filed #342 to continue working on it.

**Pull request merged**